### PR TITLE
Add Link Shortening Domain Certificate examples

### PR DESCRIPTION
**Pull request by bld010**

### CODE REVIEW

File: messaging/link-shortening/link-shortening-domain-cert/link-shortening-domain-cert.9.x.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import com.twilio.Twilio;
+import com.twilio.rest.messaging.v1.DomainCerts;
+
+public class Example {
+    // Find your Account SID and Auth Token at twilio.com/console
+    // and set the environment variables. See http://twil.io/secure
+    public static final String ACCOUNT_SID = System.getenv("TWILIO_ACCOUNT_SID");
+    public static final String AUTH_TOKEN = System.getenv("TWILIO_AUTH_TOKEN");
+
+    Sring certAndPrivateKey = """-----BEGIN CERTIFICATE-----
```
Comment (jeffge926) [inline messaging/link-shortening/link-shortening-domain-cert/link-shortening-domain-cert.9.x.java:12]: Just curious, it has 3 quotation marks  (""") Is this expected ?

Comment (bld010) [reply messaging/link-shortening/link-shortening-domain-cert/link-shortening-domain-cert.9.x.java:12]: Ah, this is the JDK15 "text blocks" feature. Is there a better, preferred way to deal with multiline strings in Java?

File: messaging/link-shortening/link-shortening-domain-cert/link-shortening-domain-cert.1.x.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
+// Download the helper library from https://www.twilio.com/docs/go/install
+package main
+
+import (
+	"fmt"
+
```
[the remark this answers is not in the document]
Comment (bld010) [reply messaging/link-shortening/link-shortening-domain-cert/link-shortening-domain-cert.1.x.go:6]: Some code editors organize imports this way (standard imports, empty line, then third-party imports). Doesn't affect functionality; just a stylistic choice.

File: messaging/link-shortening/link-shortening-domain-cert/link-shortening-domain-cert.9.x.java (hunk 1 above)
Comment (jeffge926) [reply messaging/link-shortening/link-shortening-domain-cert/link-shortening-domain-cert.9.x.java:12]: Ah that makes sense!

**Review by sarahcstringer — APPROVED** [review]

**Pull request merged**